packages/babel-preset-react-app/create.js

```diff
@@ -130,7 +130,7 @@ module.exports = function(api, opts, env) {
       require('@babel/plugin-syntax-dynamic-import').default,
       isEnvTest &&
         // Transform dynamic import to require
-        require('babel-plugin-transform-dynamic-import').default,
+        require('babel-plugin-dynamic-import-node').default,
```

Comment:
the `.default` shouldn't be here; https://unpkg.com/babel-plugin-dynamic-import-node@2.2.0/lib/index.js uses add-module-exports.

Reply:
`.default` is actually not needed anywhere in presets/plugins, `babel` handles those.